nbs/60_medical.imaging.ipynb

```diff
@@ -415,7 +415,7 @@
     {
```

Comment:
<p>I'd remove this output - it's a lot of space that doesn't say much</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2772/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='99'/>

Reply:
<p>got it, removed</p>